apple-pie/styles/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+    margin-top: 0%;
+   
+}
+
+.background-image { 
+    background-image: url(/apple-pie/images/apple-pie.jpg);
+    background-size:cover;
+    height: 75vh;
+    position: relative;
+    background-position: 0 0;
+    
+}
+.title {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: 5vw;
+    color: white;
+    width: 100vw;
+    height:50vh;
+
+}
+
+p {
+    margin-top: 5%;
+    text-align: center;
+    font-size: 25px;
+    width: 100%;
+    height: 100px;
+    
+} 
+
+.info-image {
+    align-items: center;
+    width: 30%;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+    margin-bottom: 30px;
+}
+
+hr {
+    width: 90%;
+    height: 25;
+    border: 1px solid grey;
+}
+
+.unorganized-list {
+    list-style-type: none;
+    font-size: 18px;
+    font-weight: lighter;
+    
+
+}
+
+.unorganized-list li {
+    margin:5px;
+}
+
+
+.recipe-info {
+    align-items: center;
+    width: 30%;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+    margin-bottom: 30px;
+}
+
+h3 {
+    margin: 3% 2%;
+    font-size: 20px;
+    font-weight:lighter;
+} 
+
+.organized-list {
+    list-style-type: none;
+}
+
+.organized-list li {
+    font-size: 18px;
+    width: 50%;
+} 
```

Comment:
You can add some margins at the left so your text will be a little more centered. So at the end of the file you can add this:

```
hr{
    margin-left: 100px;
}

h3, ol, ul{
    margin-left: 150px;
}
```